static/styles/compose.css

```diff
@@ -109,11 +109,24 @@
             position: relative;
             margin: 0 5px;
         }
+
+        button.narrow_to_compose_recipients {
+            background: transparent;
+            font-size: 18px;
+            padding: 1px;
+            line-height: 20px;
+            opacity: 0.6;
+            border: 0;
+
+            &:hover {
+                opacity: 1;
+            }
+        }
     }
 
     .pm_recipient {
         margin-left: 5px;
-        margin-right: 35px;
+        margin-right: 52px;
         display: flex;
```

Comment:
We don't need this extra margin if we are not showing the button for pms.

Reply:
Fixed. Forgot to remove this while removing previous pms specific changes.